sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10286,7 +10285,7 @@
               "allowCalculateOverride": false,
               "validate": {
                 "required": true,
-                "onlyAvailableItems": false,
```

Comment:
There is a radio at line 10223 (custodyOfChildWelfare) missing the property

Reply:
That line should be connected to the change at line 10218.